darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -2093,6 +2094,201 @@ def test_catboost_model_with_eval_set(self, lgb_fit_patch):
         assert lgb_fit_patch.call_args[1]["eval_set"] is not None
         assert lgb_fit_patch.call_args[1]["early_stopping_rounds"] == 2
 
+    def test_fit_predict_with_categorical_features(self):
+        """Test a RegressionModel with categorical features"""
+        target_df = pd.DataFrame(
```

Comment:
there is a bit of code duplication for target_df, past_cov_df, and their time series creation throughout the tests.
Can we try reduce this?

We can for example save the time series in the Test Class itself and then just access them in the tests.

Reply:
Good point. I've created the property `inputs_for_tests_categorical_covariates` for this. Does that make sense to you?